packages/cli/test/utils/mocking.js

```diff
@@ -4,16 +4,16 @@ import Command from '../../src/command'
 jest.mock('../../src/imports', () => {
   return {
     core: jest.fn().mockImplementation(() => ({
-      Nuxt: () => {}
+      Nuxt: function () {}
```

Comment:
@pimlie Just curious why this was needed? :D

Reply:
It wasnt really, except maybe for a clearer (future) understanding of the code. Arrow functions can't be used as a constructor, as `Nuxt` etc are mocks for class constructors I thought it would be better to explicitly indicate that.
Also the only reason it wasnt needed because there are no tests which use these standard mocks directly, all dev tests use mockNuxt and mockBuilder to explicitly set an implementation.